test/v2/models/request/amendHistoricNonFhlUkPiePeriodSummary/AmendHistoricNonFhlUkPiePeriodSummaryFixtures.scala

```diff
@@ -170,4 +170,72 @@ trait AmendHistoricNonFhlUkPiePeriodSummaryFixtures {
        |    "deductions": $downstreamJsonExpensesConsolidated
        |} 
        |""".stripMargin)
```

Comment:
As these fixtures are used across multiple places, we should consider moving them out of the `models` folder and into `test/fixtures`